images/installer/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN ln -s /usr/bin/pip3.11 /usr/bin/pip3
 RUN pip3 install --no-cache-dir boto3 botocore ansible-core
 
 # ansible-galaxy will not be available until ansible-core is installed
-RUN ansible-galaxy collection install -p /usr/share/ansible/collections amazon.aws && \
+RUN ansible-galaxy collection install -p /usr/share/ansible/collections amazon.aws community.general && \
```

Comment:
we're going to drop these usages of `community.general` from this pr right as we are no longer using this module?